filament/backend/include/private/backend/MetalPlatform.h

```diff
@@ -17,28 +17,36 @@
 #ifndef TNT_FILAMENT_DRIVER_PLATFORM_METAL_H
 #define TNT_FILAMENT_DRIVER_PLATFORM_METAL_H
 
-#include <stdint.h>
-
 #include <backend/DriverEnums.h>
 #include <backend/Platform.h>
 
+#import <Metal/Metal.h>
+
 namespace filament {
 namespace backend {
 
 class MetalPlatform : public DefaultPlatform {
 public:
     ~MetalPlatform() override;
-};
 
-} // namespace backend
-
-class PlatformMetal final : public backend::MetalPlatform {
-public:
-    backend::Driver* createDriver(void* sharedContext) noexcept override;
+    Driver* createDriver(void* sharedContext) noexcept override;
     int getOSVersion() const noexcept override { return 0; }
-    ~PlatformMetal() override;
+
+    virtual id<MTLDevice> createDevice() noexcept;
+    virtual id<MTLCommandQueue> createCommandQueue(id<MTLDevice> device) noexcept;
+
+    /**
+     * Obtain a MTLCommandBuffer enqueued on this Platform's MTLCommandQueue. The command buffer is
+     * guaranteed to execute before all subsequent command buffers.
```

Comment:
Does "subsequent command buffers" refer to the command buffers created by the method, or the ones created privately in the driver?  Or both?

Reply:
Both. Good question, I'll update the comment.